heroku_deploy.sh

```diff
@@ -45,6 +45,7 @@ if [ ${STATUS_CODE} -eq 404 ]; then
   heroku create $HEROKU_APP_NAME >>${LOGFILE} 2>>${LOGFILE}
   if [ $? -ne 0 ]; then
     print_danger "Failed to create Heroku app.\n"
+    rm -rf app
```

Comment:
This was added because if the heroku deployed failed then in subsequent deploy to ghpages, contents of app directory was being pushed.